tests/test_cas.py

```diff
@@ -252,9 +252,9 @@ def test_select_returns_feature_structures(cas_with_collections_xmi, typesystem_
     typesystem = load_typesystem(typesystem_with_collections_xml)
     cas = load_cas_from_xmi(cas_with_collections_xmi, typesystem=typesystem)
 
-    arrs = list(cas.select("cassis.StringArray"))
+    arrs = list(cas.select("uima.cas.StringArray"))
```

Comment:
We dont need the call to `list` anymore I think, driveby fix pls?

Reply:
Done